tox.ini

```diff
@@ -29,11 +29,12 @@ basepython =
     py39: python3.9
     py38: python3.8
     py37: python3.7
-    py36: python3.6
 
 [testenv:py38-docs]
 changedir = docs/source
-deps = -rdocs/requirements.txt
+deps =
+    Django>=3.2,<3.3
```

Comment:
Why you didn't add this to the `docs/requirements.txt`?

Reply:
Seems like all the auditlog package is needed (and installed with tox by default).